scripts/release.sh

```diff
@@ -5,6 +5,7 @@
 #
 set -e
 
+ARCH=$(go env GOARCH)
```

Comment:
this won't work for mastering builds from a single machine-- it'll need to iterate over the supported archs